Branch-SDK-TestBed/src/io/branch/branchandroiddemo/MainActivity.java

```diff
@@ -196,7 +197,7 @@ public void onClick(View v) {
 	@Override
 	protected void onStart() {
 		super.onStart();
-		branch = Branch.getInstance(this.getApplicationContext(), "5680621892404085");
+		branch = Branch.getInstance(this.getApplicationContext(), "76039613931586025");
```

Comment:
Oops both of those are leftover from my android redirect testing. Nice
catch!!

On Friday, January 2, 2015, qinweigong <notifications@github.com> wrote:

> In Branch-SDK-TestBed/src/io/branch/branchandroiddemo/MainActivity.java
> <https://github.com/BranchMetrics/Branch-Android-SDK/pull/20#discussion-diff-22428335>
> :
>
> > @@ -196,7 +197,7 @@ public void onClick(View v) {
> >  	@Override
> >  	protected void onStart() {
> >  		super.onStart();
> > -		branch = Branch.getInstance(this.getApplicationContext(), "5680621892404085");
> > +		branch = Branch.getInstance(this.getApplicationContext(), "76039613931586025");
>
> new app key because of the change of package name?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BranchMetrics/Branch-Android-SDK/pull/20/files#r22428335>
> .
>